App.js

```diff
@@ -1,13 +1,15 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-
+import Dog from './Dog'
 
 
 export default function App() {
   return (
     <View style={styles.container}>
       <Text>Welcome to my App!!!</Text>
```

Comment:
Nice clean solution! Feel free to merge you changes into main when ready.

Reply:
Awesome, thanks!